modules/async_worker.py

```
@@ -494,7 +494,8 @@ def apply_control_nets(async_task, height, ip_adapter_face_path, ip_adapter_path
             cn_img = resize_image(HWC3(cn_img), width=width, height=height)
 
             if not async_task.skipping_cn_preprocessor:
-                cn_img = preprocessors.canny_pyramid(cn_img, async_task.canny_low_threshold,
+                #using standard canny instead of pyramid canny because we switched from lora canny to xinsir canny
```

Flake8 (7.2.0)

[error] block comment should start with '# '

(E265)